lib/csv_helper/loader.rb

```diff
@@ -19,9 +19,22 @@ def load(file, options = {})
     def load_records(file, options)
       records = []
 
-      records = klass == Institution ? load_csv_with_version(file, records, options) : load_csv(file, records, options)
+      records = if klass == Program
+                  load_csv_with_row(file, records, options)
+                elsif klass == Institution
+                  load_csv_with_version(file, records, options)
+                else
+                  load_csv(file, records, options)
+                end
```

Comment:
This is looking like a good candidate for `case`
```ruby
records = case klass
          when Program
             load_csv_with_row(file, records, options)
          when Institution
             load_csv_with_version(file, records, options)
          else
             load_csv(file, records, options)
          end
```

Reply:
has to be `case name` and then use `.name` for when statements
changed